transform/mattermost-analytics/models/marts/web_app/platform/_platform__models.yml

```diff
@@ -0,0 +1,74 @@
+version: 2
+
+models:
+  - name: eg_performance_events
+    description: |
+      Events Group performance_events table containing metrics 
+      from web-app and web-app RC telemetry for category = performance.
+
+    columns:
+      - name: _source_relation
```

Comment:
Tests are missing.

Reply:
I will have this ready by tomorrow morning. Will be adding some basic ones to begin